openrndr-jvm/openrndr-gl3/build.gradle.kts

```diff
@@ -52,6 +52,7 @@ kotlin {
             dependencies {
                 runtimeOnly(project(":openrndr-jvm:openrndr-gl3-natives-windows"))
                 runtimeOnly(project(":openrndr-jvm:openrndr-gl3-natives-macos"))
+                runtimeOnly(project(":openrndr-jvm:openrndr-gl3-natives-macos-arm64"))
```

Comment:
This was necessary to build the project on arm64 macOS. There is also a `linux-arm64` project. I didn't add it yet, because I can't test if it's necessary or not. I'd guess, it is.